cli/lib/kontena/cli/stacks/yaml/reader.rb

```diff
@@ -368,7 +368,7 @@ def interpolate(content, use_opto: true, substitutions: {}, raise_on_unknown: fa
               if use_opto
                 opt = variables.option(var)
                 if opt.nil?
-                  to_env = variables.find { |opt| opt.to[:env][var] }
```

Comment:
I think this used to mostly work accidentally when `to: env: string` because of `String#[]` but could have produced some invalid results:

```ruby
"foofoo"["oof"]
=> "oof"
```

Reply:
And it also produced no such method `[]` for NilClass when the option had no `to[:env]`